### ISSUE TITLE
Unable to Access Sickchill website due to No username/password

### ISSUE THREAD

**Issue by MasterCtrl** [SickChill/SickChill#6036]
Branch/Commit: master
OS: Synology
Browser: Firefox

Some troubleshooting was suggested in #6020 but i am still not able to access the website.
I updated on the 27th and was met with 
![image](https://user-images.githubusercontent.com/2587101/75513954-2c4d2b00-5a5b-11ea-98a9-9af5be6712f4.png)
Talking with miigotu i have edited the config.ini(location sourced from the sickchill logs so it is correct) and added values to the web_username and web_password fields after first stopping the sickbeard-custom package in synology. inspecting the config file both before and while the package is running the settings stay correct but as soon as i stop the package in synology the web_password field becomes blank.
Additional steps i have tried, Complete reinstall of the sickchill/sickbeard package using both the sickbeard-custom method and BenjV's synology package then repeating the steps above, and using different username password combinations.

Any help would be appreciated.

**Comment by miigotu** [comment]
@neoatomic isn't there a different place the real config.ini is stored and he is editing a cached version or something like that?

**Comment by neoatomic** [comment]
No, they use some symlinking of folders, but the original files should be stored here :
/volume1/@appstore/sickbeard-custom/var/config.ini

Access to the Syno's filesystem has gotten a little complicated from DSM6 and above.
So smartest way to make changes to the config.ini is to make a backup from within the SickChill GUI.
Then open the zipfile and edit the config.ini. Than restore that modified zip backup trough the SC GUI.

Another (old) issue can be if there is still a "restore" folder left in that location.
So basically SC restores that every time SC starts, but doesn't delete it afterwards.
I think we fixed that in the past with a PR, but might be good to check if that folder is present.

For support/troubleshooting i have always been in favor to add an option in the SC GUI to remove the "cache files". (with leftovers of restore folders)
That might solve those odd issues that you normally would spent a lot of support time on.

**Comment by MasterCtrl** [comment]
Same result unfortunately, clean reinstall from the sickbeard-custom package. restore of my settings zip with a modified config.ini with web_username and web_password set appropriately and im still met with the no password webpage.

**Comment by neoatomic** [comment]
Only additional thing i can think of is if the entry's of web_username and web_password are already in the config.ini. So you basically are adding them twice.
Or the location inside the config.ini file, they might need to be placed in a specific section/location in the file.
Otherwise I'm out of ideas....

**Comment by miigotu** [comment]
Remove the password/username, access it from the same network, update through the UI, set the password through the UI.

Not going to be removing the required login

**Comment by MasterCtrl** [comment]
Some additional info as i just got this working, i have a non-standard ip address range setup on my router (31.41.59.*) and this caused sickchill to think i am connecting remotely. I also believe the password wasnt sticking because i have encryption_version = 1

**Comment by miigotu** [comment]
You should pick an ip range that is in spec with the local range set aside by ip4 specification, glad it works now.

**Comment by viperidae** [comment]
It's a little bit of a pain where you have a reverse proxy already providing authentication.
I now have to set a username/password and add basic auth headers in the proxy config

**Comment by miigotu** [comment]
@viperidae if you are using basic auth for your reverse proxy, just set the same user/pass in SC and it will pass through
